src/System.Runtime/tests/System/DecimalTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Tests
 {
+    [Collection("CultureTestCollection")]
```

Comment:
> I wanted to avoid using RemoteInvoke where it's not 100% necessary.

That's very difficult.  Lots of operations implicitly depend on the current culture: you can't just look to see where CurrentCulture is explicitly being used.  That's why we've explicitly put modifications of CurrentCulture into a separate process.